dkc/core/tests/test_authorized_uploads.py

```diff
@@ -0,0 +1,179 @@
+from datetime import timedelta
+
+from django.utils import timezone
+import pytest
+
+from dkc.core.models import AuthorizedUpload
+
+
+@pytest.fixture
+def authorized_upload(admin_api_client, folder):
+    return admin_api_client.post('/api/v2/authorized_uploads', data={'folder': folder.id})
+
+
+@pytest.mark.django_db
+def test_create_authorized_upload(admin_api_client, folder):
+    resp = admin_api_client.post('/api/v2/authorized_uploads', data={'folder': folder.id})
+    assert resp.status_code == 201
+    assert resp.data['folder'] == folder.id
+    assert resp.data['creator'] == admin_api_client.handler._force_user.id
+    assert 'signature' in resp.data
+
+
+@pytest.mark.django_db
+def test_authorized_upload_requires_write_access(api_client, public_folder, user):
+    api_client.force_authenticate(user)
```

Comment:
The cookiecutter has [an `authenticated_api_client` fixture](https://github.com/girder/cookiecutter-django-girder/blob/3e416a831e953ec927760cf8a017fd71df199716/%7B%7B%20cookiecutter.project_slug%20%7D%7D/%7B%7B%20cookiecutter.pkg_name%20%7D%7D/%7B%7B%20cookiecutter.first_app_name%20%7D%7D/tests/conftest.py#L18), which for some reason is missing from DKC-next. It might be helpful to add, for this kind of case.

The `user` should be shared between the fixture and this function.

Reply:
A side note about these fixtures: it would be great if there was an easier way to grab the user out of `admin_api_client` and `authenticated_api_client`; it's possible now but you have to reach into a private symbol. No big deal as long as we're OK with that practice.